tests/phpunit/data/blocks/fixtures/core__columns__deprecated.server.html

```diff
@@ -1,5 +1,5 @@
 
-<div class="wp-container-1 wp-block-columns has-3-columns">
+<div class="is-layout-flex wp-container-1 wp-block-columns has-3-columns">
```

Comment:
Does this keep `wp-container-1` because it has non-default flex settings?

Reply:
Yes, columns set `no-wrap` so it has a non-default setting from the perspective of the flex Layout type. Thanks for double-checking!